test/jasmine/tests/draw_newshape_test.js

```diff
@@ -1426,6 +1426,43 @@ describe('Activate and edit editable shapes', function() {
 
             .then(done, done.fail);
         });
+
+        it('should be possible to drag shapes referencing non-categorical axes', function(done) {
+            Plotly.newPlot(gd, {
+            data: [
+                {
+                x: ["2015-02-01", "2015-02-02", "2015-02-03"],
+                y: [14, 17, 8],
+                mode: "line",
+                },
+            ],
+            layout: {
```

Comment:
Please specify a `width` and a `height` for the graph.
FYI - The test didn't pass on my machine.